app/assets/javascripts/analytics.js

```diff
@@ -0,0 +1,29 @@
+//Piwik analytics shim
+//data is sent to https://analytics.brave.com
+
+<!-- Piwik -->
+
+  var _paq = _paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//analytics.brave.com/";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', '6']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+  })();
+
+<!-- End Piwik Code -->
+
+
+//trying to track if a user clicked on the verify button for a Wordpress flow
```

Comment:
better if you move this comment below directly next to the `if (target.dataset.trackWordpress === 1){`